src/client/utils/dom.js

```diff
@@ -346,8 +350,8 @@ export function isHammerheadAttr (attr) {
            attr.indexOf(INTERNAL_ATTRS.storedAttrPostfix) !== -1;
 }
 
-export function isIframe (el) {
```

Comment:
This is a breaking change. But `testcafe` doesn't use changed functions. Will we increase the `hammerhead` major version because of this?
/cc @inikulin